pages/linux/gpasswd.md

```diff
@@ -0,0 +1,23 @@
+# gpasswd
+
+> Administer /etc/group and /etc/gshadow. Administer linux group.
+
+- Define group administrotor(s), only for system administrotor:
```

Comment:
Typo: `administrotor` -> `administrator`. 

What do you mean by `only for system administrator`? If it's that you need to be root to execute it, we usually indicate this by prefixing the command with `sudo`.